mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapper.java

```diff
@@ -172,49 +172,48 @@ public MailboxId create(Mailbox mailbox) throws MailboxException {
 
         CassandraId cassandraId = CassandraId.timeBased();
         mailbox.setMailboxId(cassandraId);
-        if (!tryCreate(mailbox, cassandraId).block()) {
+        if (!tryCreate(mailbox, cassandraId)) {
```

Comment:
I prefer enforcing block in the final method.

This makes `a single mapper call SHOULD not call several time block` review easier.